pkg/metrics/server.go

```diff
@@ -1,25 +1,32 @@
 package metrics
 
 import (
+	"bytes"
 	"context"
+	"crypto/md5"
+	"errors"
 	"fmt"
+	"io"
 	"net/http"
+	"os"
+	"time"
 
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"gopkg.in/fsnotify.v1"
 
 	"k8s.io/klog/v2"
 )
 
 const (
-	tlsCRT = "/etc/secrets/tls.crt"
-	tlsKey = "/etc/secrets/tls.key"
+	certsDir = "/etc/secrets"
+	tlsCRT   = certsDir + "/tls.crt"
+	tlsKey   = certsDir + "/tls.key"
```

Comment:
Could you use filepath.Join instead ( https://pkg.go.dev/path/filepath#Join )?

Reply:
I had this thought, however we can't use a function call to set a constant.